bin/install_python3_5.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
@ntodosiychuk  btw:I found out that we can completely replace bash with ansible, at least for Python installation:)
I will fill in pull request this week end. 
spoiler:D :
```
    tasks:
      - name: get&unpack Python
        sudo: yes
        sudo_user: "{{ lookup('env', 'USER') }}"
        unarchive:
          src: https://www.python.org/ftp/python/3.5.1/Python-3.5.1.tar.xz
          dest: {{ app_dir }}
          remote_src: True

      - name: compile Python
        sudo: yes
        sudo_user: "{{ lookup('env', 'USER') }}"
        command: chdir={{ app_dir }}/Python-3.5.1 {{ item }}
        with_items:
          - ./configure
```